src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/SystemCommandLine/Utils.tt

```diff
@@ -0,0 +1,444 @@
+            {
+                // Generate Option
+                if (data.HasShortcut)
+                {
+#>
+<#= Indent(indent) #>var <#= optionNameVar #> = new Option<<#= data.DataType #>>(aliases: new[] { "<#= data.ShortcutName #>", "<#= data.FullName #>" }, description: <#= data.Help #>);
+<#+
+                }
+                else
+                {
+#>
+<#= Indent(indent) #>var <#= optionNameVar #> = new Option<<#= data.DataType #>>(name: "<#= data.FullName #>", description: <#= data.Help #>)
+<#+
+                }
+#>
+<#= Indent(indent) #>{
+<#= Indent(indent) #>    Arity = <#= data.OptionType #>,
+<#= Indent(indent) #>};
+<#+
+                // End Generate option
+            }
+#>
+<#= Indent(indent) #><#= commandVarName #>.Add(<#= optionNameVar #>);
+<#+
+        }
+    }
+
+    void GenerateHandlerArguments(XElement optionsNode, int indent = 4)
+    {
+        // Create paramethers for command handler
+        bool isFirst = true;
+        var paramEnumerator = optionsNode.Elements().GetEnumerator();
+        while (paramEnumerator.MoveNext())
+        {
+            XElement option = paramEnumerator.Current;
+            if (!isValidOption(option))
+            {
+                continue;
+            }
+            var data = new OptionData(option);
+#>
+<#= Indent(indent) #><#= getSeparator(isFirst) #><#= StringUtilities.GetParamName(data.Name) #>
+<#+
+            isFirst = false;
+        }
+    }
+
+    string GenerateCommandHandlerFunctionArgs(XElement optionsNode)
+    {
+        var sb = new StringBuilder();
+        var paramEnumerator = optionsNode.Elements().GetEnumerator();
+        while (paramEnumerator.MoveNext())
+        {
+            XElement option = paramEnumerator.Current;
+            if (!isValidOption(option))
+            {
+                continue;
+            }
+            var data = new OptionData(option);
+
+            sb.Append($", {GenerateOptionNameVar(data)}");
+        }
+
+        return sb.ToString();
+    }
+
+    void GenerateCommandHandler(string commandNameVar, string verbNounPrefix, XElement noun, int indent = 4)
+    {
+#>
+<#= Indent(indent) #>// Create handler delegate handler for <#= commandNameVar #>
+<#= Indent(indent) #><#= commandNameVar #>.SetHandler((
+<#+
+        GenerateHandlerArguments(noun, indent + 4);
+#>
+<#= Indent(indent) #>) =>
+<#= Indent(indent) #>{
+<#= Indent(indent) #>    var args = new <#= verbNounPrefix #>Args()
+<#= Indent(indent) #>    {
+<#+
+        // Generate option arg assignment
+        foreach (XElement option in noun.Elements())
+        {
+            if (!isValidOption(option))
+            {
+                continue;
+            }
+
+            var data = new OptionData(option);
+#>
+<#= Indent(indent) #>        <#= data.FormalName #> = <#= StringUtilities.GetParamName(data.Name) #>,
+<#+
+        } // end foreach option arg assignment
+#>
+<#= Indent(indent) #>    }; // end of args assignment
+
```

Comment:
Your generated files have a new line character in the first line. I *think* it may be this line causing it. Having a hard time eyeballing it, but this doesn't appear to have a control block so likely this.

Reply:
Removed changes on VerbArgs template and generated code. I can live without these changes for now.